CMakeLists.txt

```diff
@@ -441,6 +450,50 @@ endif()
 add_subdirectory(smtk)
 
 ################################################################################
+# Install a package targets file
+#
+# Setup the exports for the library when used from an installed location
+message(STATUS "MOAB_INSTALL_TARGETS: ${MOAB_INSTALL_TARGETS}")
```

Comment:
I would remove or at least comment this out. It might be useful for debugging but will confuse newcomers to SMTK.